code/controllers/subsystem/SSshuttles.dm

```diff
@@ -277,4 +277,17 @@ SUBSYSTEM_DEF(shuttle)
 
 	QDEL_LIST_CONTENTS(remove_images)
 
+/datum/controller/subsystem/shuttle/proc/mail_never_fails()
```

Comment:
I love the proc name in concept, but can you rename it to something that better describes what it does?

Reply:
Changed to mail_delivery.